packages/react-hooks/src/useInflation.ts

```diff
@@ -7,35 +7,49 @@ import type { Inflation } from './types';
 
 import { useEffect, useState } from 'react';
 
-import { getInflationParams } from '@polkadot/apps-config';
-import { BN_MILLION, BN_ZERO } from '@polkadot/util';
+import { getInflationParams, UniformEraPayoutInflationParams } from '@polkadot/apps-config';
+import { BN_BILLION, BN_MILLION, BN_ZERO } from '@polkadot/util';
+import { BN_THOUSAND } from '@polkadot/util/bn/consts';
 
 import { createNamedHook } from './createNamedHook';
 import { useApi } from './useApi';
 import { useCall } from './useCall';
 
 const EMPTY: Inflation = { idealInterest: 0, idealStake: 0, inflation: 0, stakedFraction: 0, stakedReturn: 0 };
 
+function calcInflationUniformEraPayout (totalIssuance: BN, inflationParams: UniformEraPayoutInflationParams): number {
```

Comment:
here we just need `inflationParams.yearlyInflationInTokens` instead of whole `inflationParams`